.github/workflows/golang_lint_and_test.yml

```diff
@@ -9,13 +9,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
-        with:
-            go-version: '1.17'
       - run: go version
       - uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be
           # specified without patch version: we always use the latest patch version.
           # https://github.com/golangci/golangci-lint/releases
-          version: v1.42.0
-      - run: go test $(go list ./... | grep -v /sbom | grep -v /rsa_cipher)
+          version: v1.39
+      - run: go test $(go list ./... | grep -v /sbom | grep -v /rsa_cipher)
```

Comment:
The ⛔️ means that GitHub expects the files should end with one and only on carriage return.